### DISCUSSION TITLE
Playtime : importing existing tracks in columns.

### DISCUSSION THREAD

**Discussion by Nightflying60** [helgoboss/helgobox#1067] [Ideas]
Is there a plan for being able to do that?

**Comment by helgoboss** [comment]
You can already do that on a column-by-column basis by setting the playback track.

Other than that, there's already a feature request like this which is being implemented as we speak. So I will close this one as duplicate.

**Comment by Nightflying60** [comment]
Ho, sorry for doing the wrong thing 😕.
I'm a bit confused with all the information available.
Didier

**Comment by helgoboss** [comment]
No problem, you didn't do the wrong thing. Duplicate issues are a common thing when software grows. Here's the issue I was talking about before: #1062
